devicetypes/smartthings/fibaro-rgbw-controller.src/fibaro-rgbw-controller.groovy

```diff
@@ -600,271 +271,38 @@ def refresh() {
  */
 def updateZwaveParam(params) {
 	if ( params ) {
-        def pNumber = params.paramNumber
-        def pSize	= params.size
-        def pValue	= [params.value]
-        log.debug "Updating ${device.displayName} parameter number '${pNumber}' with value '${pValue}' with size of '${pSize}'"
+		def pNumber = params.paramNumber
+		def pSize	= params.size
```

Comment:
What is with the tab between `pSize` and `=`

Reply:
looked better in my editor, lol